x86/proofs/decode.ml

```diff
@@ -258,7 +258,8 @@ let read_VEXP = new_definition `read_VEXP (p:2 word) =
 let read_VEX = define
  `(!l. read_VEX T l =
    read_byte l >>= \(b,l). bitmatch b with [r:1; v:4; L; p:2] ->
-   SOME((SOME(word_zx (word_not r)), VEXM_0F, word_not v, L, (F, Rep0, SG0)), l)) /\
+   SOME((SOME(word_shl (word_zx (word_not r)) 2), 
```

Comment:
Previously the REX.R bit is not set in the 2th location of the half-byte. The added shift fix that.